src/components/Header/Navbar.jsx

```diff
@@ -2,50 +2,48 @@
 
 import Tenant from "@/lib/tenant/tenant";
 import { usePathname } from "next/navigation";
-import { HStack } from "../Layout/Stack";
 import { HeaderLink } from "./HeaderLink";
 import styles from "./header.module.scss";
 
 export default function Navbar() {
   const pathname = usePathname();
-  const tenant = Tenant.current();
+  const { ui } = Tenant.current();
+
+  const hasProposals = ui.toggle("proposals") && ui.toggle("proposals").enabled;
+  const hasProposalsHref = Boolean(ui.page("proposals")?.href);
 
   return (
-    <HStack className={styles.main_nav}>
-      {tenant.ui.toggle("proposals") &&
-        tenant.ui.toggle("proposals").enabled && (
-          <HeaderLink
-            href="/"
-            isActive={pathname.includes("proposals") || pathname === "/"}
-          >
-            Proposals
-          </HeaderLink>
-        )}
+    <div className={`flex flex-row ${styles.main_nav}`}>
+      {hasProposals && (
+        <HeaderLink
+          href={hasProposalsHref ? ui.page("proposals")?.href : "/"}
+          target={hasProposalsHref ? "_blank" : "_self"}
+          isActive={pathname.includes("proposals") || pathname === "/"}
```

Comment:
Enabling our nav to work with external links. Not loving this but Ether.fi needs to link to proposal snapshot until our governor is in place.